### PR TITLE
update_healthcheck

### PR DESCRIPTION
**Pull request by borovHokage**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/248
Remove UseHealthChecks()
Map healthcheck endpoint to '/{serviceID}/hc'
Add RabbitMQ healthcheck Add SQL Server healthcheck
Rework healthcheck endpoint response
create pull request & fix bugs

### CODE REVIEW

File: src/MessageService/Startup.cs [hunk 1]
```diff
@@ -23,27 +24,40 @@
...
+
+            _rabbitMqConfig = Configuration
+                .GetSection(BaseRabbitMqConfig.SectionName)
+                .Get<RabbitMqConfig>();
         }
 
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddKernelExtensions();
+            //services.AddKernelExtensions();
```
Comment (BelehovEgor) [inline src/MessageService/Startup.cs:60]: comment???

Comment (KostinP) [inline src/MessageService/Startup.cs:60]: Remove

**Review by KostinP — APPROVED** [review]
Update appsettings.json

**Review by KostinP — CHANGES_REQUESTED** [review]
Mistake

File: src/MessageService/Startup.cs [hunk 2]
```diff
@@ -55,11 +69,18 @@ public void ConfigureServices(IServiceCollection services)
...
 
+            services.AddHttpContextAccessor();
+
             services.AddDbContext<MessageServiceDbContext>(options =>
             {
                 options.UseSqlServer(connStr);
             });
 
+            services
+                .AddHealthChecks()
```
Comment (BelehovEgor) [inline src/MessageService/Startup.cs:80]: you have AddHealthChecks above, delete

File: Common.Configurations.targets [hunk 3]
```diff
@@ -33,11 +33,11 @@
   </ItemGroup>
 
   <ItemGroup Condition="$(AssemblyName.Contains('.UnitTests')) != 'true'">
-    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.8" />
+    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.21" />
   </ItemGroup>
 
   <ItemGroup Condition="$(AssemblyName.Contains('.UnitTests'))">
-    <PackageReference Include="LT.DigitalOffice.UnitTestKernel" Version="1.0.1" />
+    <PackageReference Include="LT.DigitalOffice.UnitTestKernel" Version="1.0.2" />
```
Comment (KostinP) [inline Common.Configurations.targets:40]: 1.0.4

**Review by BelehovEgor — CHANGES_REQUESTED** [review]

File: Common.Configurations.targets [hunk 4]
```diff
@@ -33,11 +33,11 @@
   </ItemGroup>
 
   <ItemGroup Condition="$(AssemblyName.Contains('.UnitTests')) != 'true'">
-    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.8" />
+    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.21" />
```
Comment (KostinP) [inline Common.Configurations.targets:36]: 1.1.22

**Review by KostinP — CHANGES_REQUESTED** [review]

File: src/MessageService/Startup.cs (hunk 1 above)
Comment (borovHokage) [reply src/MessageService/Startup.cs]: done

Comment (borovHokage) [reply src/MessageService/Startup.cs]: done

File: src/MessageService/Startup.cs (hunk 2 above)
Comment (borovHokage) [reply src/MessageService/Startup.cs:78]: done

**Review by 24eeeee — APPROVED** [review]

**Review by BelehovEgor — APPROVED** [review]

File: Common.Configurations.targets (hunk 3 above)
Comment (TIove) [inline Common.Configurations.targets:40]: Up to date version is 1.0.4

**Review by KostinP — APPROVED** [review]

File: Common.Configurations.targets (hunk 3 above)
Comment (BelehovEgor) [reply Common.Configurations.targets:40]: it will be in refactoring, not now

**Review by TIove — APPROVED** [review]

**Pull request merged**